addons/alerting.libsonnet

```diff
@@ -70,6 +70,18 @@ function(config) {
   local infoRoute = trimRoutesTmpl(std.manifestYamlDoc(infoRouteTmpl, quote_keys=false)),
   local allRoutes = std.stripChars(criticalRoute + '\n' + teamRoutes + '\n' + warningRoute + '\n' + infoRoute, '[]'),
 
+
+  // This was a nightmare to do. The URL that we get from the query that triggered the alert is the address of the prometheus
+  // Since we can't access it from outside, we want to link to grafana with the query
+  // So we descent into madness, i.e. we use regex to replace the relevant parts and concatenate that with the url of our grafana. In order of execution
+  // "reReplaceAll "%22" "%5C%22" (index .Alerts 0).GeneratorURL - .GeneratorURL is a link to the prometheus with the query. We replace %22 (", with (\"), as this is what makes grafana happy
+  // reReplaceAll ".*expr=" ... - The URL is something like: http://prometheus.:9090/graph?g0.range_input=1h&g0.expr=QUERY&g0.tab=1 We keep everything after `expr=` and replace the rest with our grafana URL
+  // reReplaceAll "&g0.tab=1" "%22%7D%5D%7D" - replaces the last bit (&g0.tab=1) with ("}]})
+  // reReplaceAll `\\+` "%20" - replace all (+) with spaces ( ) - again to make grafana happy
+  // reReplaceAll "%0A" "" - newlines with nothing
+  // reReplaceAll "%28" "(" | reReplaceAll "%29" ")" - self explanatory
```

Comment:
I really appreciate your time explaining all the logic behind this very complex template, but maybe we could keep comments very objective to what they intend to do, which is just explain the logic 🙂.

Could we remove things that do not bring value to the comment, like:
* This was a nightmare to do
* So we descent into madness

`// reReplaceAll "%28" "(" | reReplaceAll "%29" ")"` doesn't feel self-explanatory to me, at least not at first sight 😅. Took me some time to understand that we're replacing those special characters with `(` and `)`. Now that I understood it does seem pretty self-explanatory but it did took me some time 😬.

What I wanted to get with the paragraph above was:
* If it really is self-explanatory, I believe there is no need to add such comment and we could just remove it
* If explaining might save others' time trying to understand it, maybe we could add an explanation to that comment